arraylist/src/main/java/ua/hillel/ArrayList.java

```diff
@@ -1,68 +1,113 @@
 package ua.hillel;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 /**
  * Implement a list that is based on array. This resizable data structure
  * is based on an array and is a simplified version of {@link java.util.ArrayList}.
  */
 public class ArrayList<T> implements List<T> {
+    private Object[] elements;
+    private int size;
+
+    private void createNewArray() {
```

Comment:
Rename to `resizeIfNeeded`, please, as `createNewArray` is not really give an understanding of why it is needed.